### PR TITLE
Assignment2 mods

### PR DESCRIPTION
**Pull request by CarlyLanglois**
for autograding

### CODE REVIEW

File: src/assignments/scrabble-scorer.rst [hunk 1]
```diff
@@ -22,50 +22,173 @@ Requirements
...
    program.
 
+Starter Code
+------------
+
+You only need to pay attention to one file here, ``scrabble-scorer.js``. Within this JavaScript
+file is still some more starter code that you don't need to touch. We'll point out what you 
+should be modifying here to write your Scrabble Scorer program.
+
+Hit the repl.it run button initially and you'll see a message printed to the console:
```
Comment (clarkroman4) [inline src/assignments/scrabble-scorer.rst:47]: Currently, the repl.it starter code still needs updating, but that doesn't prevent the PR from approval.

File: src/assignments/scrabble-scorer.rst [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 
-#. The data structure used to store the letter point values is inefficient.
-#. The program should have three scoring algorithms and allow the user to pick
-   which one to use.
+Let's roll.
 
-These new features will make the program more efficient and user friendly.
+.. TODO: mod this replit instruction. 3 types of students. independent learning track, instructor led track, and independent readers
 
 If you are NOT enrolled in a repl.it classroom for this course, fork the
```
Comment (clarkroman4) [inline src/assignments/scrabble-scorer.rst:17]: Reference to repl.it classroom - should it be directing to github classroom?

File: src/assignments/scrabble-scorer.rst [hunk 3]
```diff
@@ -327,12 +373,14 @@ Bonus Missions
...
-#. Go to the Graded Assignment #2 page in Canvas and click *Submit Assignment*.
-#. Paste the URL into the Website URL input.
-#. Click *Submit Assignment* again.
-#. Notify your TA that your assignment is ready to be graded.
+.. #. From the address bar at the top of the browser window, copy the URL of the
+..    repl.it that contains your solution.
+.. #. Go to the Graded Assignment #2 page in Canvas and click *Submit Assignment*.
+.. #. Paste the URL into the Website URL input.
+.. #. Click *Submit Assignment* again.
+.. #. Notify your TA that your assignment is ready to be graded.
```
Comment (clarkroman4) [inline src/assignments/scrabble-scorer.rst:386]: "If in the Instructor-Led Track, notify your TA that your assignment is ready to be graded" or something to that extent...

**Review by clarkroman4 — COMMENTED** [review]
A couple comments regarding references to repl.it classroom.

**Comment by CarlyLanglois** [comment]
closing because this is open now on the indep. repo. 
https://github.com/LaunchCodeEducation/js-independent-track/pull/1

**Pull request closed without merging**